src/main/java/org/kairosdb/core/http/rest/MetricsResource.java

```diff
@@ -256,28 +271,35 @@ public Response add(InputStream json)
 		}
 		catch (JsonIOException e)
 		{
+			span.log(e.getMessage());
```

Comment:
Can we mult-icatch all the copypasta?

Reply:
Just a hint: @beverage is talking about this technique: https://docs.oracle.com/javase/8/docs/technotes/guides/language/catch-multiple.html

Example:
```java
catch (JsonSyntaxException|MalformedJsonException e)
{
    span.log(e.getMessage());
    JsonResponseBuilder builder = new JsonResponseBuilder(Response.Status.BAD_REQUEST);
    return builder.addError(e.getMessage()).build();
}
```